tests/test_objective_funs.py

```diff
@@ -271,7 +271,7 @@ def test(eq):
             """Ensure compilation without any errors from JAX, related to issue #625."""
             obj = ObjectiveFunction(QuasisymmetryBoozer(eq=eq))
             obj.build()
-            obj.compile()
+            obj.compile("all")
```

Comment:
why the change to all here?

Reply:
I removed the default explicit compile step so this is just to keep coverage the same (and to make sure all derivatives work)